nilearn/surface/utils.py

```diff
@@ -0,0 +1,56 @@
+"""Utilities for surface mesh, data, images."""
+
+import numpy as np
+
+
+def assert_polydata_equal(data_1, data_2):
+    """Check that 2 PolyData data are equal."""
+    set_1 = set(data_1.parts.keys())
+    set_2 = set(data_2.parts.keys())
```

Comment:
Why not create method `equals(data)` in class `PolyData` that compares the specified `data` with the instance itself ?